src/index.js

```diff
@@ -1,4 +1,13 @@
 import joke from './joke.js';
 import './styles/main.scss';
+import add from './add.js';
+joke();
```

Comment:
- [ ] i think you don't need this line and joke.js file in your project so kindly remove that one as it is unprofessional